README.md

```diff
@@ -5,9 +5,23 @@ This is a lab for investigating adding a layer of webservices to Joomla
 
 For more details join #webservices channel at https://glip.com/
 
-
 How to install
 ====
+* Install a full current Joomla somewhere
+* git clone https://github.com/joomla-projects/webservices.git OR https://github.com/chrisdavenport/webservices.git
+* Zip or tar-gzip everything in the /component directory
+* Install component package file as an extension in the usual way
+* Copy all the files from the github clone to the web root of the CMS
+* sudo composer require willdurand/Negotiation (don't know why this is needed, but it is)
```

Comment:
`composer install` installs the current status of the `composer.lock` file - so that you have exactly the same dependencies (i.e. if there are version updates it doesn't matter - you have exactly the same versions as the main git repo). So you need to ensure that this goes in the composer.lock file. Then this step won't be required